sql/ddl_api.sql

```diff
@@ -89,16 +89,14 @@ CREATE OR REPLACE FUNCTION  set_number_partitions(
     dimension_name          NAME = NULL
 ) RETURNS VOID AS '@MODULE_PATHNAME@', 'ts_dimension_set_num_slices' LANGUAGE C VOLATILE;
 
--- Drop chunks older than the given timestamp. If a hypertable name is given,
--- drop only chunks associated with this table. Any of the first three arguments
--- can be NULL meaning "all values".
+-- Drop chunks older than the given timestamp for the specific hypertable.
 CREATE OR REPLACE FUNCTION drop_chunks(
+    table_name  REGCLASS,
     older_than "any" = NULL,
-    table_name  NAME = NULL,
-    schema_name NAME = NULL,
     newer_than "any" = NULL,
     verbose BOOLEAN = FALSE,
-    cascade_to_materializations BOOLEAN = NULL
+    cascade_to_materializations BOOLEAN = NULL,
+    allows_materialization BOOLEAN = FALSE
```

Comment:
It is for internal usage, but exposed because this is how we communicate with the data node. I called it `force` originally, but was asked to change it. If you do not have a concrete suggestion for another name, I will leave it as it is.

Reply:
I don't see that this parameter is set internally. Can you point to the place? My current understanding is that users need to set it.

The parameter is exposed publicly, so the name should be meaningful to users. I don't understand why it is needed and its semantics, thus it is difficult for me to propose the name.

Reply:
Inside `src/bgw_policy/job.c`:
```
	/* Invoke drop chunks via fmgr so that the call can be deparsed and sent
	 * also to remote data nodes. */
	num_dropped = chunk_invoke_drop_chunks(table_relid,
										   older_than,
										   older_than_type,
										   args->cascade_to_materializations,
										   true);
```

Reply:
I am also confused by this new parameter. 

FWIW, my comment on `force` was for an internal function to find a hypertable. Not this public API.

I don't think this parameter was ever discussed or agreed upon. This PR was supposed to remove the wildcard option for drop_chunks, not introduce new parameters that seem unrelated to the original change.

Reply:
I fully agree with @erimatnor I don't understand why this PR should add new parameter, since it is about requiring table name in drop chunks public SQL function.

Reply:
We don't yet support caggs on distributed hypertables and haven't discussed how it should work. So, I don't think this PR should be concerned with drop_chunks on caggs.

Reply:
Removed `allow_materialization`.